grapple/actions.py

```diff
@@ -72,8 +72,7 @@ def add_app(app_label: str, prefix: str = ""):
 
     # Add snippet models to model collection.
     for snippet in get_snippet_models():
-        if snippet._meta.app_label == app_label:
-            models.append(snippet)
+        models.append(snippet)
```

Comment:
I'm not sure is this will throw any errors 🤔.

Reply:
Hey, first thanks a lot for your feedbacks, much appreciated ! 

I am not sure what was the reason of this change, I should have added some comments...but I think that when "collection" are added registered as snippet, the check on the app_label would not add snippet to model collection. 

I will start working on it next week, I currently use this branch as is on production without issues, but it may not cover all cases of snippet's usage 🤔